### PR TITLE
Small refactor

### PR DESCRIPTION
**Pull request by disconnect3d**

### CODE REVIEW

File: advancedSearch.py [hunk 1]
```diff
@@ -51,9 +51,8 @@ def final(self, _file):
             print(colored("INTERESTING FILE HAS BEEN FOUND!!!", 'cyan'))
             print(colored("The rule defined in 'rules.yaml' file has been "
                   + "triggerred. Checkout the file " + _file, 'cyan'))
-            interesting_file = True
```
Comment (disconnect3d) [inline advancedSearch.py:54]: This var is never used.

File: core.py [hunk 2]
```diff
@@ -220,8 +213,6 @@ def folder_reader(path):
                                 if decompressed:
                                     queue.put(decompressed)
 
-                                f.close()
```
Comment (disconnect3d) [inline core.py:223]: Files are closed by the exit part of context manager of file objects (see http://book.pythontips.com/en/latest/context_managers.html#context-managers).

Comment (securing) [reply core.py:223]: You're right, thank you for pointing that!

File: advancedSearch.py (hunk 1 above)
Comment (securing) [reply advancedSearch.py:54]: You're right, again thanks!

**Comment by securing** [comment]
Huge thanks @disconnected for your PRs!

**Pull request merged**